_posts/2020-05-04-report-cs-teaching-methods.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Report: A CS Teaching Methods Experiment"
+tags: ["csc696", "teaching", "pedagogy", "experiment preparation", "msu"]
+author: "Nathan Hartzler"
+---
+
+*During the Spring 2020 semester, I worked to develop an experiment to research Pedagogy in Computer Science*.
+
+# Working Hypothesis
+When teaching a computer science concept, is student understanding increased if an analogous hands-on activity is performed in addition to the traditional lecture of the concept?
+
+# Experiment Framework
+- Working with a computer science instructor, design as many activities as feasible in line with a course syllabus. The preference is for a computer science course with multiple lab sections in order to have a control group for the experiment.
+
+- Each activity should cover a specific course objective such as **Understanding Sorting Algorithms** or **Understanding Error Detection**.
+
+- In a multiple lab sections, one section should recieve a traditional lecture and programming exercise for a given topic. The other section(s) should perform an activity in addition to the lecture and programming exercise.
+
+- Non-credit, anonymized surveys should be given throughout the semester to evaluate how the activity (or lack thereof) affected student understanding of a topic.
+
+# Sample Activity - Error Detection
+
+To familiarize students with the concept of flipped bit error detection and parity bits, a simple activity is presented on [csunplugged.org](https://csunplugged.org/en/topics/error-detection-and-correction/unit-plan/parity-magic/).
+
+1. First the instructor has a volunteer student lay out a 5 x 5 grid of black and white square tiles. These are white on one side and black on the other. The student is asked to choose tile colors at random.
+
+    ![Example 5 by 5 grid](https://storage.googleapis.com/cs-unplugged.appspot.com/static/img/topics/parity-cards-6x6-grid-step-1.png)
```

Comment:
Please download the image and add it to this repo. That way we don't rely on that external website. Also, be sure to site from where these images are being sources.